src/app/components/acf/acf.html

```diff
@@ -2,7 +2,7 @@
   <div class="form-horizontal" ng-form="vm.queryForm" ng-if="vm.mode === 'select' || vm.mode === 'enter' || vm.mode === 'edit'">
     <div class="panel panel-primary">
       <div class="panel-heading">
-        <h2 class="panel-title" ng-if="vm.mode === 'select' || vm.mode === 'enter'">Login to an Alternate Care Facility (ACF)</h2>
+        <h2 class="panel-title" ng-if="vm.mode === 'select' || vm.mode === 'enter'">Select an Alternate Care Facility (ACF)</h2>
```

Comment:
I'm not sure what you mean here. Get rid of line 6:
```<h2 class="panel-title" ng-if="vm.mode === 'enter'">Create a new ACF</h2>```?
Or the entire code block for if ```vm.mode === enter```?